netsocadmin/templates/form.html

```diff
@@ -24,7 +24,7 @@
                     err.innerText = "Invalid request"
                 } else if (req.responseText !== "Available") {
                     err.innerText = "Username not available";
-                } else if (! /^[a-z0-9-_]+[a-z0-9]$/.test(uid)) {
+                } else if (! /^[a-z0-9][a-z0-9-_]+[a-z0-9]$/.test(uid)) {
```

Comment:
regex should probably be changed to the following to allow 1 or 2 character usernames `[a-z0-9][a-z0-9-_]*[a-z0-9]*`